server/lorax_server/utils/paged_attention.py

```diff
@@ -14,11 +14,14 @@
             f"Could not import vllm paged attention. Make sure your installation is correct. Complete error: {e}"
         )
 
-if torch.cuda.is_available():
-    # TODO(travis): fix for CUDA 8.9 (Lovelace)
-    fp8_supported = torch.cuda.get_device_capability()[0] >= 9 #or (torch.cuda.get_device_capability()[0] == 8 and torch.cuda.get_device_capability()[1] >= 9)
-else:
-    fp8_supported = False
+# TODO(travis): fix for CUDA 8.9 (Lovelace) and 9.0 (Hopper)
+# if torch.cuda.is_available():
+#     fp8_supported = (
+#         torch.cuda.get_device_capability()[0] >= 9
+#     )  # or (torch.cuda.get_device_capability()[0] == 8 and torch.cuda.get_device_capability()[1] >= 9)
+# else:
+fp8_supported = False
```

Comment:
Can we also add this same comment to `server/lorax_server/models/flash_causal_lm.py` or just refactor to use that?

Reply:
Not sure I follow, where would you want it added in `flash_casual_lm.py`?

Reply:
Ah, okay I see. That seems to be unused code. Removed.